src/components/Sam.tsx

```diff
@@ -1,7 +1,156 @@
+    height: 20,
+    width: 20
+  },
+  details: {
+    alignItems: 'center'
+  },
+  column: {
+    flexBasis: '33.33%'
+  },
+  helper: {
+    borderLeft: `2px solid ${
+      theme.palette.divider
+      }`,
+    padding: theme.spacing(1, 2)
+  },
+  link: {
+    color: theme.palette.primary.main,
+    textDecoration: 'none',
+    '&:hover': {
+      textDecoration: 'underline'
+    }
+  },
+  listItem: {
+    width: '100%',
+    maxWidth: 360,
+    backgroundColor: theme.palette.background.paper
+  },
+  inputField: {
+    width: '30ch',
+  },
+  spacedBox: {
+    float: 'right',
+  },
+}));
 
-const Sam = () => {
-  return (<div>Sam</div>);
-};
+export default function DetailedExpansionPanel() {
+  const classes = useStyles();
 
-export default Sam;
+  return (
+    <Grid container spacing={3}>
+      <div className={
+        classes.root
+      }>
+        <Box paddingTop={2} p={3}>
+        <ExpansionPanel defaultExpanded>
+          <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}
+            aria-controls="panel1c-content"
+            id="panel1c-header">
+            <div className={
+              classes.column
+            }>
+              <Typography className={
+                classes.heading
+              }>CourseFlo</Typography>
+            </div>
+          </ExpansionPanelSummary>
+          <ExpansionPanelDetails>
+            <List component="nav"
+              className={
+                classes.listItem
+              }
+              aria-label="profile navigation">
+              <ListItem button>
+                <ListItemText primary="Browse all courses" />
+              </ListItem>
+              <Divider />
+              <ListItem button>
+                <ListItemText primary="View my courses" />
+              </ListItem>
+              <Divider />
+              <ListItem button>
+                <ListItemText primary="Customize my profile" />
+              </ListItem>
+              <Divider />
+              <ListItem button>
+                <ListItemText primary="Logout" />
+              </ListItem>
+            </List>
+          </ExpansionPanelDetails>
+        </ExpansionPanel>
+        </Box>
+      </div>
+      <div className={classes.column} />
+      <div className={classes.root}>
+              <Card>
+                <CardActions className={classes.inputField} >
+                  <Typography className={classes.heading}>Name</Typography>
+                  <Grid container justify="flex-end" >
+                    <TextField className={classes.spacedBox}  id="standard-basic" label="Give us a name!" />
+                  </Grid>
+                </CardActions>
+                <CardActions className={classes.inputField}>
+                  <Typography className={classes.heading}>Degree</Typography>
+                  <Grid container justify="flex-end" >
+                    <TextField  className={classes.spacedBox} id="standard-basic" label="What are you studying?" />
```

Comment:
This placeholder text overflows and is cutoff by the line of the textbox. Perhaps shorten or better yet make the textboxes longer, especially for longer names and degrees.

Reply:
Thanks for catching that, it does not show up on my screen: 
![image](https://user-images.githubusercontent.com/44733905/83941858-8e78a300-a7a3-11ea-8068-072d0e36b677.png)